salt/utils/zeromq.py

```diff
@@ -82,6 +82,8 @@ def ip_bracket(addr):
     Convert IP address representation to ZMQ (URL) format. ZMQ expects
     brackets around IPv6 literals, since they are used in URLs.
     '''
-    if addr and ':' in addr and not addr.startswith('['):
-        return '[{0}]'.format(addr)
+    _ip_addr = ipaddress.ip_address(addr)
```

Comment:
The `ipaddress` module isn't imported for use here.

Reply:
Oh, right. And it also should be imported from `_compat`, not directly!

Reply:
It's also missing from the `salt/transport/zeromq.py` file.